settings.py

```diff
@@ -5,6 +5,7 @@
 
 REPOSITORY = os.getenv("INPUT_REPOSITORY")
 GH_TOKEN = os.getenv("INPUT_GH_TOKEN")
+GH_API_URL = os.getenv("GITHUB_API_URL")
```

Comment:
Update the tests with this new env var, probably just adding the `github.com` in the mocks, don't need to add new ones if you don't want. Any problems with the tests sent a message that I help you

Reply:
Hi, the GITHUB_API_URL env. var is a default variable and automatically added to each action. I think it's not needed to add it. It's automatically set by the github action based on the used github.